condor.go

```diff
@@ -382,14 +383,16 @@ func main() {
 	launcher := New(cfg)
 
 	uri := cfg.GetString("amqp.uri")
+	exchangeName := cfg.GetString("amqp.exchange.name")
+	exchangeType := cfg.GetString("amqp.exchange.type")
```

Comment:
This is avery minor suggestion, but it might be nice to have a common place where these configuration settings are defined so that we don't have the configuration setting names referenced in multiple places.